agg/SafeList.cs

```diff
@@ -77,6 +135,11 @@ public void SetParent(T parent)
 			this.parentItem = parent;
 		}
 
+		public ReadOnlyList<T> ReadOnly()
```

Comment:
Can we describe the benefit and use case for ReadOnly function and how does it differ from SafeList in general?

Reply:
Removing